internal/store/postgres/asset_repository.go

```diff
@@ -348,6 +385,49 @@ func (r *AssetRepository) DeleteByURN(ctx context.Context, urn string) error {
 	return nil
 }
 
+func (r *AssetRepository) DeleteByQueryExpr(ctx context.Context, queryExpr queryexpr.ExprStr) ([]string, error) {
+	var urns []string
+	err := r.client.RunWithinTx(ctx, func(tx *sqlx.Tx) error {
+		query, err := queryexpr.ValidateAndGetQueryFromExpr(queryExpr)
+		if err != nil {
+			return err
+		}
+
+		urns, err = r.deleteByQueryAndReturnURNS(ctx, query)
+		if err != nil {
+			log.Printf("Failed to delete by query expr: %v", err)
```

Comment:
please don't use this `log` package

Reply:
removed, already logged in deleteByQueryAndReturnURNS func and service layer